README.md

```diff
@@ -29,25 +19,45 @@ Once you've written your provider, you'll want to [publish it on the Terraform R
 go install
 ```
 
-## Adding Dependencies
+## Configuring the provider
 
-This provider uses [Go modules](https://github.com/golang/go/wiki/Modules).
-Please see the Go documentation for the most up to date information about using Go modules.
+The provider can be configured by modifying the following variables in `config/config.go`. When building a version of the provider for distribution, all of these should be hard-coded and not set with environment variables.
 
-To add a new dependency `github.com/author/dependency` to your Terraform provider:
+- `OpenAPIPath`: The URI where your OpenAPI spec lives. This can also be set via the `AEP_OPENAPI` environment variable.
```

Comment:
could this point to a local file? I guess the idea of *not* having network connectivity for a Terraform provider might be asinine, but I was thinking about how do we bundle the full terraform provider (including the dependent openapi file) into a distribution so it's like any other provider. A local file (relative to the go code) might work well.

Reply:
Since we're using aep-lib-go, we actually get that for free. I'll update the docs to reflect.